pkg/karmadactl/cmdinit/utils/rbac.go

```diff
@@ -0,0 +1,89 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+)
+
+// ClusterRoleFromSpec ClusterRole spec
+func ClusterRoleFromSpec(name string, rules []rbacv1.PolicyRule) *rbacv1.ClusterRole {
```

Comment:
Sorry, I don't understand the meaning of the function name `FromSpec`.

Reply:
I just moved it from the original place, and did not change the name, what do you suggest to change?

Reply:
Yeah, I find it. Suggest `ClusterRoleFromSpec` -> `ClusterRoleFromRules`.

Reply:
Change it to `ClusterRoleFromSpec`, how about that?

Reply:
fixed. @XiShanYongYe-Chang